test/test_torch.py

```diff
@@ -190,7 +190,7 @@ def unary_eval(symbol, x):
 
 @pytest.mark.parametrize('dims', [(), ('a',), ('a', 'b')])
 @pytest.mark.parametrize('symbol', [
-    '~', '-', 'abs', 'sqrt', 'exp', 'log', 'log1p',
+    'abs', 'sqrt', 'exp', 'log', 'log1p',
```

Comment:
Why aren't we testing `~` and `-` any more?

Reply:
My bad, I was debugging something, and forgot to reinstate this.